app/build.gradle

```diff
@@ -53,14 +53,14 @@ def fabricPropertiesFile = rootProject.file(".configs/fabric.properties")
 fabricProperties.load(new FileInputStream(fabricPropertiesFile))
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion 26
```

Comment:
Yeah... I'm not sure what is the best practice for this. Whatever you think that works.